### PR TITLE
[CN-930] Add K8s auto discovery test workflow

### PR DESCRIPTION
**Pull request by SeriyBg**
Add K8s auto discovery test workflow

Fixes CN-930

Checklist:
[x] Labels (`Team:`, `Type:`, `Source:`, `Module:`) and Milestone set
[x] Label `Add to Release Notes` or `Not Release Notes content` set
[x] Request reviewers if possible
[x] Send backports/forwardports if fix needs to be applied to past/future releases
[x] New public APIs have `@Nonnull/@Nullable` annotations
[x] New public APIs have `@since` tags in Javadoc

### CODE REVIEW

**Comment by SeriyBg** [comment]
run-lab-rub

**Comment by SeriyBg** [comment]
run-lab-rub

**Comment by SeriyBg** [comment]
\run-lab-run

**Comment by SeriyBg** [comment]
run-lab-run

**Review by cagric0 — APPROVED** [review]

File: .github/workflows/k8s-terraform-integration-tests.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          credentials_json: ${{ env.GCLOUD_KEYFILE_JSON }}
+
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@v1
+        with:
+          install_components: 'gke-gcloud-auth-plugin,kubectl'
+
+      - name: Copy kubeconfig
+        run: |
+          echo ${KUBECONFIG}
```
Comment (cheels) [inline .github/workflows/k8s-terraform-integration-tests.yml:69]: Please remove 'echo' from here as well :)

**Review by cheels — APPROVED** [review]

**Pull request merged**